annotatedcsv/reader.go

```diff
@@ -142,25 +145,14 @@ func (r *Reader) NextRow() bool {
 	return true
 }
 
-// Row returns the values in the current row of the current section.
+// Row returns the raw values in the current row of the current section.
```

Comment:
I wonder if this should fill in default values rather than returning them empty.
WDYT?

Reply:
Good idea 👍